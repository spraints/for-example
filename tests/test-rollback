
copyrev: e22446297cdefafc1a83cdc7fd347c5190b11a56
copy: tests/test-undo

#!/bin/sh

mkdir t
cd t
hg init
echo a > a
hg add a
hg commit -m "test" -d "1000000 0"
hg verify
hg parents
hg status
hg rollback
hg verify
hg parents
hg status
