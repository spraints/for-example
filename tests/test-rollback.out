
copyrev: 24df4008ffbf597e8060c44abf07ed4be22a4d89
copy: tests/test-undo.out

checking changesets
checking manifests
crosschecking files in changesets and manifests
checking files
1 files, 1 changesets, 1 total revisions
changeset:   0:0acdaf898367
tag:         tip
user:        test
date:        Mon Jan 12 13:46:40 1970 +0000
summary:     test

rolling back last transaction
checking changesets
checking manifests
crosschecking files in changesets and manifests
checking files
0 files, 0 changesets, 0 total revisions
A a
